Source/Blazorise/Switch.razor.cs

```diff
@@ -37,6 +40,20 @@ protected override void BuildClasses( ClassBuilder builder )
 
         protected override string TrueValueName => "true";
 
+        [Parameter]
+        public Color Color
+        {
+            get
```

Comment:
Do get as one line `get { return color; }`. It's just a convention in Blazorise :)